patroni/api.py

```diff
@@ -252,8 +252,8 @@ def get_postgresql_status(self, retry=False):
     def get_tags(self):
         return {'tags': self.server.patroni.tags}
 
-    def log_message(self, format, *args):
-        logger.debug("API thread: " + format % args)
+    def log_message(self, fmt, *args):
+        logger.debug("API thread: %s - - [%s] %s", self.client_address[0], self.log_date_time_string() + fmt % args)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Not enough arguments for logging format string](https://www.codacy.com/app/cyberdemn/patroni/file/2034615145/issues/source?bid=3127594&fileBranchId=3129207#l256)